blog-post.html

```diff
@@ -1,12 +1,175 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <title>Blog Post</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel='stylesheet' type="text/css">
+  <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-  </body>
+ 
+<body>
+  <div class="container-blog-post">
+    <div class="item header">
+      <div class="header-wrapper">
+        <a href="index.html"><h1>POST ONLY</h1></a>
+        <h4 id="h4-section">A PAGE WITH JUST A BLOG CONTENT.</h4>
+      </div>
+      <div class=separation-content></div>
+    </div>
+    <div class="content-3">
+      <div>
+        <h3><span>THIS IS A</span><span id="dark"> BLOG POST</span><span> ONLY LAYOUT</span></h3>
+      </div>
+      <div><img class="image-blog-post" src="https://placekitten.com/408/287" title="Another Kitten" alt="Another Kitten">

```

Comment:
I'm sure you noticed, but the `<img>` tag here isn't quite centered; it stretches too far to the left.  take a look at resizing the wrapper div or getting rid of the wrapper altogether and resizing the `<img class="image-blog-post"` itself.  setting its width to take up the amount of grid-space that the `<div class="header-wrapper">` is a good way to go about it.  But not the only way!